wiki/Article_Styling_Criteria/en.md

```diff
@@ -63,11 +83,9 @@ tags:
 ---
 ```
 
-You should only need to include words that aren't found in the article.
-
-#### Outdated and tagged pages
+##### Outdated and tagged articles
 
-*Note for translators: Do delete the `outdated` mark, but do not delete the tags.*
+*Note: Translators, do not delete the `outdated` mark, but do not delete the tags.*
```

Comment:
I think the old language makes more sense, because it tells people who aren't translators that they can safely ignore this part

if you use the new line, change `do not` to `do`

Reply:
Alright, changed a similar occurrence to match this.